packages/test/test-version-utils/src/compatConfig.ts

```diff
@@ -281,9 +283,13 @@ export function isOdspCompatCompliant(config: CompatConfig): boolean {
 }
 
 // Helper function for genCrossClientCompatConfig().
-function genCompatConfig(createVersion: string, loadVersion: string): CompatConfig {
+function genCompatConfig(
+	createVersion: string,
+	loadVersion: string,
+	deltas: string[],
```

Comment:
nit: Maybe just take in the two delta strings individually. With having an array but only expecting 2 values, there are chances of things going wrong.